pkgs/tools/text/sgml/linuxdoc-tools/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, lib, makeWrapper, fetchFromGitLab, openjade, gnumake, perl, flex,
+gnused, coreutils, which, opensp, groff, texlive, texinfo, withLatex ? false }:
+
+stdenv.mkDerivation rec {
+  version = "0.9.82";
+  pname = "linuxdoc-tools";
+
+  src = fetchFromGitLab {
+    rev = version;
+    owner = "agmartin";
+    repo = "linuxdoc-tools";
+    sha256 = "17v9ilh79av4n94vk4m52aq57ykb9myffxd2qr8kb8b3xnq5d36z";
+  };
+
+  outputs = [ "out" "man" "doc" ];
+
+  configureFlagsArray = [
+    ("--enable-docs=txt info lyx html rtf" + lib.optionalString withLatex " pdf")
+  ];
+
+  LEX="flex";
+
+  postInstall = ''
+    wrapProgram $out/bin/linuxdoc \
+      --prefix PATH : "${lib.makeBinPath buildInputs}:$out/bin" \
```

Comment:
I'm not sure to see why this is necessary, is it due to the PERL5LIB thing?

Reply:
I don't like the look of this either. It seems to both require certain binaries in it's path as well as some of the perl libraries which it ships. The perl libraries seem to be installed in /share/linuxdoc-tools for some reason, which doesn't seem right to me.

Reply:
I reduced the amount of things added to the path, but I couldn't fix the rest.